tests/functional/kvpy/tombspan.py

```diff
@@ -18,6 +18,78 @@ def check_keys(cursor: hse.KvsCursor, expected: List[bytes]):
     for x, y in zip(expected, actual):
         assert x == y
 
+def test_basic(kvdb: hse.Kvdb, kvs: hse.Kvs):
+    kvs.put(b"a", b"1")
```

Comment:
Would increase readability probably if you dropped the b prefix. Not a big deal since both are accepted though.

Reply:
Right. I skip it for any new tests I'm writing, but we may want to do this for all the older tests.

Reply:
Maybe it can just be a maint monday ticket in the future.